README.md

```diff
@@ -5,36 +5,48 @@ Loop end-to-end automated tests using `detox`
  - [Background reading](https://hackernoon.com/detox-gray-box-end-to-end-testing-framework-for-mobile-apps-196ccd9564ce)
 
 
-## Build
-- `npm run build`
+## Loop Build
+Save the current build (if any) and fetch the new one based on the branch name
 
-## Test
+### Fetch new build
+- `source ./scripts/build_loop.dev.sh --fetch-new build-289`
```

Comment:
Why does the `build_loop.dev.sh` script need to be sourced (rather than just ran)? Is there some environment variable that needs to be exported to the current shell?